### PR TITLE
perf(ming-omni): optimize CFM tail kernels

### PR DESCRIPTION
**Pull request by YzXiao101**
## Checklist

- [ ] Format your code according with pre-commit.
- [ ] Add unit tests.
- [ ] Update documentation / docstrings / example tutorials as needed.
- [ ] Provide throughput / latency benchmark results and accuracy evaluation results as needed.
- [ ] For reviewers: If you haven't made any contributions to this PR and are only assisting with merging the main branch, please remove yourself as a co-author when merging the PR.

## CI

CI runs on self-hosted GPU runners and requires a maintainer to add the
`run-ci` label. Once labeled, every subsequent push re-triggers CI as
long as the label remains. Use `/tag-and-rerun-ci higgs` or
`/tag-and-rerun-ci moss` or `/tag-and-rerun-ci qwen3-tts` to select a TTS CI
model, and
`/tag-and-rerun-ci fun-asr`, `/tag-and-rerun-ci qwen3-asr` or
`/tag-and-rerun-ci whisper-asr` to select an ASR CI model. One selector from
each family can be combined, for example `/tag-and-rerun-ci moss fun-asr`.
Draft PRs are skipped even if labeled.

### CODE REVIEW

**Comment by YzXiao101** [comment]
## Update: Ming-Omni CFM Tail Kernel Optimization

Update:
- Added a joint rotary-position kernel, fused RMSNorm, and packed QKV projection to the Ming-Omni CFM tail.
- The full 10-step CFM CUDA Graph latency decreased from **12.0451 ms to 4.4153 ms** (**63.34% lower, 2.73× faster**).
- Profiling confirmed that the optimized kernels are dispatched by both streaming and non-streaming Talker paths.

Benchmark: the baseline is `9cc50dc4`; the candidate includes all three optimizations above. All runs used 5×H200, Thinker TP4 with a separate Talker GPU, 48 English texts, concurrency 1/8, and two reversed-order rounds. All 384 formal requests completed successfully.

Mode | Concurrency | QPS Change | Audio Throughput | Mean Latency
--- | ---: | ---: | ---: | ---:
Streaming | 1 | +2.11% | +2.63% | -2.06%
Streaming | 8 | -1.95% | -0.15% | +2.64%
Non-streaming | 1 | **+52.37%** | **+55.00%** | **-34.37%**
Non-streaming | 8 | **+83.48%** | **+87.67%** | **-45.23%**

Mode | Concurrency | Baseline WER | Optimized WER | Difference
--- | ---: | ---: | ---: | ---:
Streaming | 1 | 1.777% | 1.623% | -0.155 pp
Streaming | 8 | 0.986% | 0.607% | -0.379 pp
Non-streaming | 1 | 1.468% | 2.473% | +1.005 pp
Non-streaming | 8 | 0.759% | 0.683% | -0.076 pp
Overall | — | 1.2443% | 1.3400% | +0.0957 pp

Result:
- Non-streaming QPS improved by **52.37–83.48%**, consistently across both execution orders.
- Streaming showed no repeatable end-to-end gain. The optimized LLM+CFM producer takes about **8.5 ms/patch**, while incremental `token2wav`/VAE takes about **43–46 ms/patch** and remains the limiting stage.
- Overall WER changed by **+0.0957 pp**; the non-streaming c1 cell requires further validation.

File: tests/unit_test/ming_tts/test_packed_qkv.py [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+            )
+        )
+        expected_weights.append(weight)
+        expected_biases.append(bias)
+
+    MingTTSSGLangModel.load_weights(owner, checkpoint)
+
+    torch.testing.assert_close(projection.weight, torch.cat(expected_weights))
+    torch.testing.assert_close(projection.bias, torch.cat(expected_biases))
+    assert owner._weight_load_report.loaded_shards == {
```
Comment (luojiaxuan) [inline tests/unit_test/ming_tts/test_packed_qkv.py]: [P2] load_weights assigns self.weight_load_report, but this assertion and the one on line 81 read owner._weight_load_report, so both success cases raise AttributeError after loading. Please use owner.weight_load_report.loaded_shards in both assertions.
